symbolication/datadog_uploader.go

```diff
@@ -53,14 +58,27 @@ func NewDatadogUploader() (Uploader, error) {
 		return nil, fmt.Errorf("failed to parse URL: %w", err)
 	}
 
+	uploadCache, err := lru.NewSynced[libpf.FileID, struct{}](binaryCacheSize, libpf.FileID.Hash32)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create cache: %w", err)
+	}
+
 	return &DatadogUploader{
 		ddAPIKey:  ddAPIKey,
 		intakeURL: intakeURL,
+
```

Comment:
minor: why the extra lines (mostly curious)

Reply:
mostly cosmetic to separate fields that are based on user input, from internal fields that are initialized by the structure itself